rocketpy/simulation/flight.py

```diff
@@ -3433,24 +3433,42 @@ def to_dict(self, include_outputs=False):
             "time_overshoot": self.time_overshoot,
             "name": self.name,
             "equations_of_motion": self.equations_of_motion,
+            "solution": self.solution, ##
```

Comment:
What are these `#` comments? 

```suggestion
            "solution": self.solution,
```

Reply:
Those were to keep track of the new data to be stored, I forgot to remove it